src/components/Beneficiaries/index.tsx

```diff
@@ -366,10 +368,12 @@ class BeneficiariesComponent extends React.Component<Props, State> {
     };
 
     private handleClickToggleAddAddressModal = () => () => {
-        const { memberLevels, userData } = this.props;
+        const { memberLevels, userData, beneficiaries } = this.props;
 
         if (memberLevels && (userData.level < memberLevels.withdraw.minimum_level)) {
             this.handleToggleFailModal();
+        } else if (beneficiaries && beneficiaries.length >= 10) {
+            this.props.alertPush({ message: ['error.beneficiaries.max10.addresses'], type: 'error'});
```

Comment:
Better to dispatch sendError, because it handles itself the error you passed to it